katas/content/multi_qubit_gates/index.md

```diff
@@ -196,6 +196,18 @@ The $CZ$ gate is also self-adjoint: applying it a second time reverses its effec
     ]
 })
 
+
```

Comment:
Please move this exercise to the lesson on Multi-controlled gates, per my comment in https://github.com/microsoft/qsharp/pull/1389#pullrequestreview-2013016417